src/test/resources/entity-mapping/sample-report-with-details.json

```diff
@@ -273,24 +273,28 @@
   ],
   "staffInvolved": [
     {
+      "sequence": 0,
```

Comment:
shouldn't make a difference (I don't think) but aren't sequences starting from `1` now to match NOMIS?

Reply:
apparently not according to #235